src/lib/assets/table.scss

```diff
@@ -29,6 +29,8 @@
             display: block;
             overflow-y: scroll;
```

Comment:
I know that this is not part of your changes, but could we change 
```  scss
overflow-y
```
and
```  scss
overflow-x
```
to ```  scss
overflow
```
?

Reply:
I believe we can) Thanks.